nbgrader/nbextensions/assignment_list/assignment_list.js

```diff
@@ -413,7 +432,15 @@ define([
                     },
                     type : "POST",
                     dataType : "json",
-                    success : $.proxy(that.on_refresh, that),
+                    success : function (data, status, xhr) {
+                        if (!data.success) {
+                            that.submit_error(data);
+                            button.text('Sumbit');
+                            button.removeAttr('disabled');
+                        }
+                        //$.proxy(that.on_refresh, that);
+                        //GRRRR
```

Comment:
@jhamrick I am a little bit stuck with this. How do I incorporate this refresh when the submission is successful?